### PR TITLE
Delete mscorlib facade

### LINKED ISSUE #7607: Move mscorlib facade from CoreCLR to CoreFX
cc: @jkotas @weshaggard 

Today we version mscorlib and System.Private.CoreLib as netstandard1.0 which means that we use the same two dlls across different frameworks (i.e. netcoreapp1.0 and netcoreapp1.1). This started being an issue since PR #7578 caused the mscorlib facade to type forward some types to System.IO. The problem with that, is that in netcoreapp1.0 world, System.IO.dll implementation type-forwards System.IO.Stream to mscorlib.dll facade, which used to type-forward it System.Private.CoreLib.dll where the implementation lives, but that is not the case anymore, and it now type-forwards it back to System.IO causing a type not found.

```
Resolution of System.IO.Stream type
netcoreapp1.1: System.IO(v.4.2.0.0) -> System.Runtime(v.4.2.0.0) -> System.Private.CoreLib (type lives here)
netcoreapp1.0: System.IO(v.4.1.0.0) -> mscorlib -> System.IO(v.4.1.0)....
```

This causes:

```
Unhandled Exception: System.TypeLoadException: Could not load type 'System.IO.Stream' from assembly 'System.IO, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
```

This would get fixed if we version the mscorlib facade

### PR DESCRIPTION
**Pull request by jkotas**
Fixes #7607

### CODE REVIEW

File: src/mscorlib/System.Private.CoreLib.csproj [hunk 1]
```diff
@@ -445,6 +444,7 @@
   <ItemGroup>
     <Compile Include="$(BclSourcesRoot)\Internal\Runtime\Augments\EnvironmentAugments.cs" />
     <Compile Include="$(BclSourcesRoot)\Internal\Runtime\Augments\RuntimeThread.cs" />
+    <Compile Include="$(BclSourcesRoot)\Internal\Console.cs" />
```
Comment (danmosemsft) [inline src/mscorlib/System.Private.CoreLib.csproj]: why did you rename this back (?)

Comment (jkotas) [inline src/mscorlib/System.Private.CoreLib.csproj]: I have moved it to internal namespace instead. It makes switching between the internal version and the official version a bit easier.
